pkg/queue/prometheus_stats_reporter_test.go

```diff
@@ -78,29 +83,30 @@ func TestNewPrometheusStatsReporter_negative(t *testing.T) {
 			config,
 			revision,
 			"",
+			1 * time.Second,
```

Comment:
Please add a test which has a different setting here and verify it still returns the correct numbers.

Reply:
@markusthoemmes Added tests

Reply:
Sure, I added. Please let me know what you think.